### ISSUE TITLE
Valid Quantized Mesh Header?

### ISSUE THREAD

**Issue by malaretv** [heremaps/tin-terrain#49]
I'm new to this stuff so apologies if this is user error or misunderstanding...

The coordinate values in the quantized mesh header seem off. The quantized-mesh spec says coordinate values should be in earth-centered fixed coordinates. Given that, I'd expect the magnitude of the center point to be a value close to the radius of the earth, but that doesn't seem to be the case.

As an example I ran `dem2tintiles` on the craterlake dem dataset.

The resulting header for tile 6/10/40.terrain looks like:
```
{
  "centerX": -13462700.917811524,
  "centerY": 5322463.153553393,
  "centerZ": 1331.456658194813,
  "minHeight": 1130.3309326171875,
  "maxHeight": 1532.5823974609375,
  "boundingSphereCenterX": -13462700.917811524,
  "boundingSphereCenterY": 5322463.153553393,
  "boundingSphereCenterZ": 1331.456658194813,
  "boundingSphereRadius": 885541.1267042665,
  "horizonOcclusionPointX": -13462700.917811524,
  "horizonOcclusionPointY": 5322463.153553393,
  "horizonOcclusionPointZ": 1532.5824073771364
}
```

which means the resulting magnitude for the `center` point is 14476633.994 which seems way off from the earth radius of 6378137

**Comment by delfrrr** [comment]
To my best knowledge it's correct coordinates
https://epsg.io/map#srs=3857&x=-13462700.917811524&y=5322463.153553393&z=4&layer=streets
in pseudo-merector projection https://epsg.io/3857
In this coordinates X ranges from -Pi*R to Pi*R which, with R=6378137 gives us -20026376.39 -20048966.10

@malaretv I hope it will help!

I'd also appreciate if you can share (in general terms) few details about your use case:
What is the industry you are in?
What is the final deliverable you are trying to achieve?

This info will help us to continue developing tin-terrain.

**Comment by delfrrr** [comment]
@malaretv yes you are right, spec requires earth centred coords; we do not support it yet and we created custom provider for Cesium to render our tiles https://github.com/heremaps/quantized-mesh-viewer/blob/master/src/map/surface-provider.js

I see that for the Moon you'd prefer earth centred over pseudo mercator 😂

If you have experience with C++ please consider contribution of  earth centred coords in tin-terrain; otherwise I will add feature to our backlog but cannot provide estimation.

**Comment by fester** [comment]
Fixed by #54
